database/fingertips-db/PostDeployment/PopulateFingertips.sql

```diff
@@ -0,0 +1,2993 @@
+INSERT [dbo].[AreaDimension] ([AreaKey], [Code], [Name], [StartDate], [EndDate]) VALUES (68449, N'nE38000261', N'Derby and Derbyshire ICB - 15M ', CAST(N'2014-01-01T00:00:00.0000000' AS DateTime2), CAST(N'2044-01-01T00:00:00.0000000' AS DateTime2))
+INSERT [dbo].[AreaDimension] ([AreaKey], [Code], [Name], [StartDate], [EndDate]) VALUES (68450, N'nE38000262', N'Northamptonshire ICB - 78H ', CAST(N'2014-01-01T00:00:00.0000000' AS DateTime2), CAST(N'2044-01-01T00:00:00.0000000' AS DateTime2))
+INSERT [dbo].[AreaDimension] ([AreaKey], [Code], [Name], [StartDate], [EndDate]) VALUES (68451, N'nE38000263', N'Greater Manchester ICB - 01Y ', CAST(N'2014-01-01T00:00:00.0000000' AS DateTime2), CAST(N'2044-01-01T00:00:00.0000000' AS DateTime2))
+INSERT [dbo].[AreaDimension] ([AreaKey], [Code], [Name], [StartDate], [EndDate]) VALUES (68768, N'K02000001', N'United Kingdom', CAST(N'2014-01-01T00:00:00.0000000' AS DateTime2), CAST(N'2044-01-01T00:00:00.0000000' AS DateTime2))
+INSERT [dbo].[AreaDimension] ([AreaKey], [Code], [Name], [StartDate], [EndDate]) VALUES (69317, N'U00000', N'West Norwich PCN', CAST(N'2014-01-01T00:00:00.0000000' AS DateTime2), CAST(N'2044-01-01T00:00:00.0000000' AS DateTime2))
+INSERT [dbo].[AreaDimension] ([AreaKey], [Code], [Name], [StartDate], [EndDate]) VALUES (69318, N'U00070', N'West Merton PCN', CAST(N'2014-01-01T00:00:00.0000000' AS DateTime2), CAST(N'2044-01-01T00:00:00.0000000' AS DateTime2))
+INSERT [dbo].[AreaDimension] ([AreaKey], [Code], [Name], [StartDate], [EndDate]) VALUES (69319, N'U00147', N'Aegis Healthcare PCN', CAST(N'2014-01-01T00:00:00.0000000' AS DateTime2), CAST(N'2044-01-01T00:00:00.0000000' AS DateTime2))
+INSERT [dbo].[AreaDimension] ([AreaKey], [Code], [Name], [StartDate], [EndDate]) VALUES (69320, N'U00203', N'Eccles & Irlam PCN', CAST(N'2014-01-01T00:00:00.0000000' AS DateTime2), CAST(N'2044-01-01T00:00:00.0000000' AS DateTime2))
+INSERT [dbo].[AreaDimension] ([AreaKey], [Code], [Name], [StartDate], [EndDate]) VALUES (69321, N'U00254', N'Blackheath And Charlton PCN', CAST(N'2014-01-01T00:00:00.0000000' AS DateTime2), CAST(N'2044-01-01T00:00:00.0000000' AS DateTime2))
+INSERT [dbo].[AreaDimension] ([AreaKey], [Code], [Name], [StartDate], [EndDate]) VALUES (69367, N'U04648', N'Brunel Health Group PCN6', CAST(N'2014-01-01T00:00:00.0000000' AS DateTime2), CAST(N'2044-01-01T00:00:00.0000000' AS DateTime2))
+INSERT [dbo].[AreaDimension] ([AreaKey], [Code], [Name], [StartDate], [EndDate]) VALUES (69708, N'U28791', N'Brunel Health Group PCN4', CAST(N'2014-01-01T00:00:00.0000000' AS DateTime2), CAST(N'2044-01-01T00:00:00.0000000' AS DateTime2))
+INSERT [dbo].[AreaDimension] ([AreaKey], [Code], [Name], [StartDate], [EndDate]) VALUES (69711, N'U29197', N'Phoenix PCN
+', CAST(N'2014-01-01T00:00:00.0000000' AS DateTime2), CAST(N'2044-01-01T00:00:00.0000000' AS DateTime2))
```

Comment:
I can't imagine this one is meant to contain a newline, is it? (Spotted because SonarQube is complaining about it)

Reply:
it is an automatically generated script - it is valid sql though so I don't think it is a problem

Reply:
I suppose it doesn't have to block this PR, but can we not just remove the newline? It's not correct data, and I thought we were trying to keep this initial data set relatively clean?

Reply:
changed :)